### ISSUE TITLE
Stitch for VSCode: Project fails to load when room has path layer with no path assigned

### ISSUE THREAD

**Issue by W4SD** [bscotch/stitch#164]
Stitch version: 1.51.1

When room has a path layer but has no path set, stitch fails to load the project:
```
rm_map_parent/rm_map_parent.yy
[
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "null",
    "path": [
      "layers",
      3,
      "pathId"
    ],
    "message": "Expected object, received null"
  }
]
```

**Comment by W4SD** [comment]
Assigning any path to layer solves the issue

**Comment by adam-coster** [comment]
Thanks for the clear report! That made it an easy fix.

The fix is live in 1.51.2+. Let me know if that doesn't resolve the problem!
